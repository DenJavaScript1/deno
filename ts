#!/usr/bin/env -S deno run
// deno-lint-ignore-file prefer-const
let message = 'Hello, Deno!';
console.log(message);