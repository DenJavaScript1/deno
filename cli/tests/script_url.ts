console.log("script_url", Deno.scriptUrl.pathname);

import "./script_url2.ts";
