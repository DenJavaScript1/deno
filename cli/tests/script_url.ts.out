script_url2 [WILDCARD]tests/script_url.ts
script_url [WILDCARD]tests/script_url.ts
